app/backend/proto/internal/jobs.proto

```diff
@@ -54,3 +54,13 @@ message GenerateEventUpdateNotificationsPayload {
   int64 occurrence_id = 2;
   repeated string updated_items = 3;
 }
+
+message GenerateEventCancelNotificationsPayload {
+  int64 cancelling_user_id = 1;
+  int64 occurrence_id = 2;
+}
+
+message GenerateEventDeleteNotificationsPayload {
+  int64 deleting_user_id = 1;
```

Comment:
I think mod deletion shouldn't broadcast info on who deleted the event. It should just say "the event was deleted by moderators", and not who did it exactly.

Reply:
Yes, I guess that is sensible - removed the user information now from any notification.